pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/Namespaces.java

```diff
@@ -838,7 +838,8 @@ public void unloadNamespaceBundle(@PathParam("property") String property, @PathP
     @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission") })
     public void splitNamespaceBundle(@PathParam("property") String property, @PathParam("cluster") String cluster,
             @PathParam("namespace") String namespace, @PathParam("bundle") String bundleRange,
-            @QueryParam("authoritative") @DefaultValue("false") boolean authoritative) {
+            @QueryParam("authoritative") @DefaultValue("false") boolean authoritative,
+            @QueryParam("unload") @DefaultValue("false") boolean unload) {
```

Comment:
Since the brokers already have the `loadBalancerAutoUnloadSplitBundlesEnabled` parameter, wouldn't this `unload` option be redundant? 

Just trigger the split and broker will decide whether to unload or not.

Reply:
That's correct but this admin-api is triggered by load-balancer and cli-tool. Sometimes we might want to just split the bundle using cli-tool without unloading it. Therefore, this query-param can give flexibility to perform accordingly.??